packages/devtools-reps/src/object-inspector/components/ObjectInspectorItem.js

```diff
@@ -163,6 +167,18 @@ class ObjectInspectorItem extends Component<Props> {
         };
       }
 
+      if (nodeHasGetter(item)) {
+        const parentNode = getParent(item);
+        const parentGripNode = parentNode && getClosestGripNode(parentNode);
+        const parentGrip = parentGripNode && getValue(parentGripNode);
```

Comment:
nit, but maybe we move this to a helper: `getParentGrip(item)`.

Reply:
sure